cartridges/bm_imgix_pd/cartridge/bm_imgix_pd.properties

```
@@ -0,0 +1,4 @@
+## cartridge.properties for cartridge bm_imgix_pd
+#Wed Aug 22 22:6:15 CEST 2021
+demandware.cartridges.bm_imgix_pd.multipleLanguageStorefront=true
+demandware.cartridges.bm_imgix_pd.id=bm_imgix_pd
```

NIT: missing newline at end of file
```suggestion
demandware.cartridges.bm_imgix_pd.id=bm_imgix_pd

```